dirty_cat/datasets/_ken_embeddings.py

```diff
@@ -0,0 +1,97 @@
+"""
+Get the Wikipedia embeddings for feature augmentation.
+"""
+import pandas as pd
+from sklearn.decomposition import PCA
+
+from dirty_cat.datasets import fetch_figshare
+
+
+def get_ken_embeddings(
+    types,
+    exclude=None,
+    pca_components=None,
+    emb_id="39142985",
+    emb_type_id="39266300",
+    suffix="",
+):
+    """Extract Wikipedia embeddings by type.
+
+    More details on the embeddings can be found on
+    https://soda-inria.github.io/ken_embeddings/.
+
+    Parameters
+    ----------
+    types : str
+        Types of embeddings to include. Write in lowercase.
+    exclude : str, default=None
+        Type of embeddings to exclude from the types search.
+    pca_components : int, default=None
+        Size of the dimensional space on which the embeddings will be projected
+        by a principal component analysis.
+        If None, the default dimension (200) of the embeddins will be kept.
+    emb_id : str
+        Figshare ID of the file containing all embeddings.
+    emb_type_id : str
+        Figshare ID of the file containing the type of embeddings.
```

Comment:
Same comment. Can it / should it, be the same list of names: when we give emb_id as a name, the emb_type_id number is automatically deduced (aka the table has three columns: the name and the two numbers)? In which case, this number becomes 'None' by default.

Reply:
I think this will be possible, yes.